tac/agents/v1/mail.py

```diff
@@ -192,12 +196,14 @@ def connect(self) -> bool:
 
         while not success:
             try:
+                print('HERE!')
```

Comment:
I think that was just a debugging print, we could remove it.

Reply:
Thanks!